js/piwik.js

```diff
@@ -1403,7 +1403,17 @@ if (typeof Piwik !== 'object') {
             },
             hasNodeCssClass: function (node, klassName)
             {
-                if (node && klassName && node.className) {
+                if(!node || !klassName) {
+                    return false;
+                }
+
+                // For modern browsers cf http://caniuse.com/#search=classList
+                if(node.classList) {
+                    return node.classList.contains(klassName);
```

Comment:
I think just to be safe we should also check for `node.classList && node.classList.contains`